File: WebService_v2/src/main/java/servlets/Mirror.java
package servlets;

import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import java.io.IOException;

public class Mirror extends HttpServlet {
    @Override
    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
        var keyParam = req.getParameter("key");
        resp.getWriter().println(keyParam);
    }
}

File: WebService_v2/src/main/java/servlets/Mirror.kt
package servlets

import java.io.IOException
import javax.servlet.http.HttpServlet
import javax.servlet.http.HttpServletRequest
import javax.servlet.http.HttpServletResponse

class Mirror : HttpServlet() {
    @Throws(IOException::class)
    override fun doGet(req: HttpServletRequest, resp: HttpServletResponse) {
        val keyParam = req.getParameter("key")
        resp.writer.println(keyParam)
    }
}